### PR TITLE
feat: arrow chevron color fix ACC-149

### PR DESCRIPTION
**Pull request by BrzozowskiPawel**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-149" title="ACC-149" target="_blank">TaskACC-149</a>  [iOS] fix chevrons in cells
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

This pull request resolves the issue with the grey/black arrow (chevron) being displayed If a cell has another list presented when clicked. 

### Solutions

The solution is to implement accessoryView set as SF Symbol with the selected colour. Also, we have deleted unnecessary cell type to make it more clear.

| iPhone 8 - English  | iPhone 8 - German  |  iPhone SE - English | iPhone SE - German  |
|---|---|---|---|
| Screenshot 2022-05-12 at 08 21 49  | Screenshot 2022-05-12 at 08 26 22  | Screenshot 2022-05-12 at 09 54 36 | Screenshot 2022-05-12 at 08 25 21  |

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsExternalScreenCellDescriptor.swift [hunk 1]
```diff
@@ -134,24 +134,46 @@ class SettingsExternalScreenCellDescriptor: SettingsExternalScreenCellDescriptor
...
-
         }
     }
 
     func generateViewController() -> UIViewController? {
         return self.presentationAction()
     }
 }
+
+extension UITableViewCell {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsExternalScreenCellDescriptor.swift:158]: There's a few lint issues to fix here. I would also suggest to move this to a separate file.

Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsExternalScreenCellDescriptor.swift]: @johnxnguyen it's all done. You can take a look once again :D

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1239,7 +1240,6 @@
 		EE01D08825ADA3D5001DB205 /* AppLockModule.Interactor.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE01D08325ADA3D5001DB205 /* AppLockModule.Interactor.swift */; };
 		EE01D08925ADA3D5001DB205 /* AppLockModule.View.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE01D08425ADA3D5001DB205 /* AppLockModule.View.swift */; };
 		EE09DE6025B24B2000B515F0 /* AppLockModuleRouterTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE09DE5E25B24B2000B515F0 /* AppLockModuleRouterTests.swift */; };
-		EE12922827E47E6D00C9CC60 /* Wire Share Extension.appex in Embed App Extensions */ = {isa = PBXBuildFile; fileRef = 168A16A91D9597C2005CFA6C /* Wire Share Extension.appex */; settings = {ATTRIBUTES = (RemoveHeadersOnCopy, ); }; };
```
Comment (agisilaos) [inline Wire-iOS.xcodeproj/project.pbxproj:1242]: This doesn't look right? @johnxnguyen

File: Wire-iOS/Sources/UserInterface/Settings/SettingWhiteChevronUITableViewExtension.swift [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+import Foundation
+import UIKit
+
+extension UITableViewCell {
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Settings/SettingWhiteChevronUITableViewExtension.swift:21]: The convention for files containing extensions is to call them after the class name + a description for the extension. For instance `UITableViewCell+DisclosureIndicator.swift`.

File: Wire-iOS.xcodeproj/project.pbxproj (hunk 2 above)
Comment (johnxnguyen) [reply Wire-iOS.xcodeproj/project.pbxproj:1242]: It's been happening for a while now, not sure why it keeps happening. But it doesn't seem harmful, just annoying.

File: Wire-iOS/Sources/UserInterface/Settings/SettingWhiteChevronUITableViewExtension.swift (hunk 3 above)
Comment (BrzozowskiPawel) [reply Wire-iOS/Sources/UserInterface/Settings/SettingWhiteChevronUITableViewExtension.swift:21]: Thanks, I will remember to use the proper file naming in the future :D

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1217,6 +1217,7 @@
 		D5F8BFF42007AC84008F8C3D /* UIView+TableViewHeaderFooterSizing.swift in Sources */ = {isa = PBXBuildFile; fileRef = D5F8BFF32007AC84008F8C3D /* UIView+TableViewHeaderFooterSizing.swift */; };
 		DB3E943A1B96EDBD00990482 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 8741CFAE1A937EDF00A56A63 /* LaunchScreen.xib */; };
 		DBF7235F1B5566B80036F4C5 /* a_lot_of_people.json in Resources */ = {isa = PBXBuildFile; fileRef = DBF7235E1B5566B80036F4C5 /* a_lot_of_people.json */; };
+		E52F448B2844E916008E11CB /* UITableViewCellDisclosureIndicator.swift in Sources */ = {isa = PBXBuildFile; fileRef = E52F448A2844E916008E11CB /* UITableViewCellDisclosureIndicator.swift */; };
```
Comment (johnxnguyen) [inline Wire-iOS.xcodeproj/project.pbxproj:1220]: Don't forget the `+`, as in `UITableViewCell+DisclosureIndicator`. ☺️

Comment (BrzozowskiPawel) [reply Wire-iOS.xcodeproj/project.pbxproj:1220]: Hahah ok sorry now I get it 😎

**Review by agisilaos — CHANGES_REQUESTED** [review]
We need to take care a few things before this is ready. When those two things are fixed we will be good to go. 

1. Revert the changes regarding the CountryTests
2. Change the name of the file @johnxnguyen mentioned

File: Wire-iOS Tests/CountryTests.swift [hunk 5]
```diff
@@ -22,9 +22,9 @@ final class CountryTests: XCTestCase {
 
     func testThatCountryFromDeviceGivesCurrentCountry() {
         // GIVEN
-        let countryFromDevice = Country.countryFromDevice!
+        let countryFromDevice = Country.countryFromDevice
 
         // WHEN & THEN
-        XCTAssertEqual(countryFromDevice.iso, "us")
+        XCTAssertEqual(countryFromDevice?.iso, "us")
```
[the remark this answers is not in the document]
Comment (BrzozowskiPawel) [reply Wire-iOS Tests/CountryTests.swift:28]: OK sure thing

**Review by johnxnguyen — APPROVED** [review]

File: Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme [hunk 6]
```diff
@@ -58,6 +58,9 @@
                <Test
                   Identifier = "ArticleViewTests/testArticleViewWithPicture()">
                </Test>
+               <Test
+                  Identifier = "ArticleViewTests/testArticleViewWithoutPicture()">
+               </Test>
```
Comment (agisilaos) [inline Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme:63]: This needs to be removed.

**Review by agisilaos — CHANGES_REQUESTED** [review]

**Review by agisilaos — APPROVED** [review]

**Pull request merged**